tests/garage/envs/wrappers/test_multi_env_wrapper.py

```diff
@@ -0,0 +1,125 @@
+"""Tests for garage.envs.wrappers.multi_env_wrapper"""
+import akro
+import numpy as np
+import pytest
+
+from garage.envs.wrappers import MultiEnvWrapper, RoundRobinStrategy, \
+    UniformRandomStrategy
+from garage.tf.envs import TfEnv
+
+# Disabling pylint warnings since class name and method names are self-explanatory # noqa: E501
```

Comment:
please remove these

Reply:
i don't think they should trigger errors under the latest pre-commit configuration